tests/kolainst/nondestructive/misc.sh

```diff
@@ -6,12 +6,12 @@ cd $(mktemp -d)
 
 # Ensure multicall is correctly set up and working.
 R_O_DIGEST=$(sha512sum $(which rpm-ostree) | cut -d' ' -f1)
-O_C_DIGEST=$(sha512sum $(which ostree-container) | cut -d' ' -f1)
+O_C_DIGEST=$(sha512sum $(which /usr/libexec/libostree/ext/ostree-container) | cut -d' ' -f1)
 if test "${R_O_DIGEST}" != "${O_C_DIGEST}" ; then
     assert_not_reached "rpm-ostree and ostree-container are not the same binary"
 fi
-ostree-container container --help > cli_help.txt
-assert_file_has_content_literal cli_help.txt 'ostree-container container <SUBCOMMAND>'
+rpm-ostree ex-container --help > cli_help.txt
```

Comment:
Perhaps I misunderstood this change, but I think this isn't exercising multicall dispatching anymore, right?

Reply:
Yes!  This is why we have code review :smile: